src/kernel/chainparams.cpp

```diff
@@ -491,6 +509,12 @@ class SigNetParams : public CChainParams {
         base58Prefixes[EXT_PUBLIC_KEY] = {0x04, 0x35, 0x87, 0xCF};
         base58Prefixes[EXT_SECRET_KEY] = {0x04, 0x35, 0x83, 0x94};
 
+        base58EncodedPrefixes[PUBKEY_ADDRESS] = {"m","n"};
+        base58EncodedPrefixes[SCRIPT_ADDRESS] = {"2"};
+        base58EncodedPrefixes[SECRET_KEY] =     {"9","c"};
+        base58EncodedPrefixes[EXT_PUBLIC_KEY] = {"tpub"};
+        base58EncodedPrefixes[EXT_SECRET_KEY] = {"tpriv"};
+
         bech32_hrp = "tb";
```

Comment:
as mentioned by @Sjors, I think we may want to leave room for Silent Payment prefixes as well: [`9219e4f` (#28201)](https://github.com/bitcoin/bitcoin/pull/28201/commits/9219e4f2a61d0d0b6590c443ceda4665a73c9a6b)

Probably nothing to do yet, just good to know about it - cc: @josibake

Reply:
In 7b71e6e056f5567152097d5c9a1a8e2f59c1b661 _Include Base58 encoded prefixes in chainparams_: this commit only adds base58 prefixes, silent payments will use bech32m.